internal/capability/caldera/caldera.go

```diff
@@ -141,8 +142,12 @@ func parseFacts(facts CalderaFacts) cacao.Variables {
 	return variables
 }
 
-func parseYamlAbility(bytes []byte) *models.Ability {
-	return &models.Ability{
-		AbilityID: string(bytes),
+func ParseYamlAbility(bytes []byte) *models.Ability {
```

Comment:
In Go, it's pretty difficult to test private methods (unit tests are not part of the package here). We could work around this by exporting the `parseYamlAbility` as well. Or we can leave this as a public method.